.idea/CollaBoard.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove these files from your pull request and add `.idea/` to `.gitignore`. In the future, make sure you're checking every file that you're committing.